### PR TITLE
EZP-30203: UDW disappears when scroll appears in Finder in folder div

### PR DESCRIPTION
**Pull request by tischsoic**
JIRA: https://jira.ez.no/browse/EZP-30203

## After
Ckc688Uk4N

YC4x7cwRqf

## Before
1FerO6VMTk

### CODE REVIEW

File: src/modules/universal-discovery/components/finder/css/finder.component.css [hunk 1]
```diff
@@ -7,7 +7,6 @@
     flex-wrap: nowrap;
     height: 100%;
     overflow: auto;
-    border-radius: 4px;
```
Comment (tischsoic) [inline src/modules/universal-discovery/components/finder/css/finder.component.css:10]: Removing this property fixes bug. It seems that it was too much for Firefox to render properly. 🤷‍♂️ 
It was actually redundant property, removing it does not change UI.

File: src/modules/universal-discovery/components/finder/finder.component.js [hunk 2]
```diff
@@ -384,7 +384,7 @@ export default class FinderComponent extends Component {
         const { locationsMap } = this.state;
 
         return (
-            <div className="c-finder" style={{ maxHeight: `${this.props.maxHeight}px` }}>
```
Comment (tischsoic) [inline src/modules/universal-discovery/components/finder/finder.component.js:387]: Removing _maxHeight_ fixes double scroll problem. Not related with bug directly.

File: src/modules/universal-discovery/components/finder/css/finder.component.css (hunk 1 above)
Comment (tischsoic) [reply src/modules/universal-discovery/components/finder/css/finder.component.css:10]: Bug was appearing when some folder had enough items for scroll to appear in _.c-finder__branch_ div. Everything seems to be related with some bug in Firefox browser which was unable to handle too many scrolls, overflows and border-radiuses.

**Review by dew326 — APPROVED** [review]

**Review by micszo — APPROVED** [review]

**Comment by micszo** [comment]
OK also after rebase. Only Travis left.

**Comment by lserwatka** [comment]
You can merge it up.

**Comment by tischsoic** [comment]
Merged up:
- master: https://github.com/ezsystems/ezplatform-admin-ui-modules/commit/bbc89f4f18aa40110d20f226fade56f7ad6d7e07

**Pull request merged**